### PR TITLE
change gatewaysource in AX record

### LINKED ISSUE #328: "gateway_source" in AX records should be "Configurable" for EG4A

### PR DESCRIPTION
**Pull request by rockspore**
fix #328

### CODE REVIEW

File: server/accesslog.go [hunk 1]
```diff
@@ -50,6 +51,10 @@ func (a *AccessLogServer) Register(s *grpc.Server, handler *Handler, d time.Dura
 	a.handler = handler
 	a.streamTimeout = d
 	a.context = ctx
+	a.gatewaySource = defaultGatewaySource
+	if a.handler.operationConfigType == "PROXY" {
```
Comment (theganyo) [inline server/accesslog.go:55]: I believe this should use the `product.ProxyOperationConfigType` constant, should it not?

File: server/accesslog_test.go [hunk 2]
```diff
@@ -121,6 +121,7 @@ func TestHandleHTTPAccessLogs(t *testing.T) {
 			envName:      extAuthzFields[headerEnvironment].GetStringValue(),
 			analyticsMan: testAnalyticsMan,
 		},
+		gatewaySource: "configurable",
```
Comment (theganyo) [inline server/accesslog_test.go:124]: Use a constant for all these instances of string "configurable"?

File: server/accesslog.go (hunk 1 above)
Comment (rockspore) [reply server/accesslog.go]: Done.

File: server/accesslog_test.go (hunk 2 above)
Comment (rockspore) [reply server/accesslog_test.go]: Ack.

**Review by theganyo — APPROVED** [review]

**Pull request merged**